.github/workflows/docs.yaml

```diff
@@ -18,10 +18,11 @@ permissions:
 
 jobs:
   build-docs:
+    name: Build Docs
```

Comment:
Adding names and consistent capitalization across steps for the different workflows